### ISSUE TITLE
Installation documentation and requirements

### ISSUE THREAD

**Issue by hervenicol** [metacloud/molecule#1363]
# Issue Type

- Documentation updates

# Molecule and Ansible details

```
$ ansible --version
ansible 2.6.0
  config file = /etc/ansible/ansible.cfg
  configured module search path = [u'/home/hnicol/.ansible/plugins/modules', u'/usr/share/ansible/plugins/modules']
  ansible python module location = /usr/lib/python2.7/site-packages/ansible
  executable location = /usr/bin/ansible
  python version = 2.7.15 (default, May 16 2018, 17:50:09) [GCC 8.1.1 20180502 (Red Hat 8.1.1-1)]
```

```
$ molecule --version
molecule, version 2.15.0
```

```
$ lsb_release -a
LSB Version:    :core-4.1-amd64:core-4.1-noarch
Distributor ID: Fedora
Description:    Fedora release 28 (Twenty Eight)
Release:        28
Codename:       TwentyEight
```

Molecule installation method :
- pip

Ansible installation method :
- pip

# Desired Behavior

All requirements should be listed in the same place.

# Actual Behaviour

Today, the sphinx doc talks about a few required packages:
* gcc python-pip python-devel openssl-devel

and the created README.rst (with molecule init scenario) talks about some others:
* docker-py

And on my machine (Fedora 28), it was not enough:
- docker-py seems to be deprecated, docker python lib is preferred.
- package python-libselinux was required as well.

# Proposed change
* Installation page should list all requirements,
* README.rst should point to official doc & repo.

**Comment by retr0h** [comment]
I don't use Fedora.  If you would like the docs updated please submit a PR.

**Comment by retr0h** [comment]
Addressed by #1367.
